### PR TITLE
An attempt to fix samples

### PR DESCRIPTION
**Pull request by ibrakap**
In this PR i tried to fix PSL1GHT's samples additionally make some changes in Makefiles.
<del>But i couldn't fix camera sample it gives me strange linker error.</del> It fixed at 74015cb .

### CODE REVIEW

File: samples/input/gemsample/source/rsxutil.c [hunk 1]
```diff
@@ -254,13 +254,13 @@ setRenderTarget (gcmContextData * context, rsxBuffer * buffer)
...
 
-  sf.depthFormat = GCM_TF_ZETA_Z16;
+  sf.depthFormat = GCM_SURFACE_ZETA_Z16; /* GCM_TF_ZETA_Z16 */
   sf.depthLocation = GCM_LOCATION_RSX;
   sf.depthOffset = depth_offset;
   sf.depthPitch = depth_pitch;
 
-  sf.type = GCM_TF_TYPE_LINEAR;
-  sf.antiAlias = GCM_TF_CENTER_1;
+  sf.type = 1; /* GCM_TF_TYPE_LINEAR */
```
Comment (shagkur) [inline samples/input/gemsample/source/rsxutil.c:262]: please use GCM_SURFACE_TYPE_LINEAR

File: samples/input/gemsample/source/rsxutil.c [hunk 2]
```diff
@@ -254,13 +254,13 @@ setRenderTarget (gcmContextData * context, rsxBuffer * buffer)
...
+  sf.antiAlias = 0; /* GCM_TF_CENTER_1 */
```
Comment (shagkur) [inline samples/input/gemsample/source/rsxutil.c:263]: GCM_SURFACE_CENTER_1

**Review by shagkur — CHANGES_REQUESTED** [review]
Looks good. See my inline comments.
Other than that Makefile cleanup is fine too (probably done according to the others, working ones?)

**Comment by ibrakap** [comment]
Thanks for reviews @shagkur . I am a newcomer for PS3 development. Finally i fixed camera sample. I was tested samples only in RPCS3 most of them works as expected but i didn't test on real PS3.

**Pull request closed without merging**